InternxtDesktop/Services/BackupsService.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BackupsService.swift
+//  InternxtDesktop
+//
+//  Created by Richard Ascanio on 1/8/24.
+//
+
+import Foundation
+import RealmSwift
+
+class BackupsService: ObservableObject {
+    @Published var foldernames: [FoldernameToBackup] = []
+    @Published var urls: [URL] = []
+    @Published var isBackupButtonEnabled: Bool = false
+
+    private func getRealm() -> Realm {
+        do {
+            return try Realm(fileURL: ActivityManager.realmURL)
```

Comment:
move the `ActivityManager.realmURL` to the `ConfigLoader` struct https://github.com/internxt/drive-desktop-macos/blob/main/Shared/Services/ConfigLoader.swift and save it in a property, this way the realmURL is not related to the ActivityManager only, and we have it centralized in a single place